pages/App.js

```diff
@@ -13,16 +13,62 @@ export default function List() {
     },
     {
       id: nanoid(),
-      text: "Hafermilch und Obst kaufen",
+      text: "die Welt retten",
+    },
+    {
+      id: nanoid(),
+      text: "Blumen gießen",
     },
   ]);
 
+  function addItem(event) {
+    event.preventDefault();
+    const inputElement = event.target.elements.newItem;
+    const newListItem = {
+      id: nanoid(),
+      text: inputElement.value,
+    };
+
+    const newListArray = [...todoItems, newListItem];
+    setTodoItems(newListArray);
+  }
+
+  function deleteItem(id) {
+    const newListArray = todoItems.filter((item) => {
+      if (item.id === id) {
+        return false;
+      } else {
+        return true;
+      }
+    });
+    setTodoItems(newListArray);
+  }
+
   return (
     <>
       <h1>Todo-List</h1>
-      <ul>
+      <form onSubmit={addItem}>
+        <input
+          type="text"
+          id="newItem"
+          name="newItem"
+          placeholder="todo hinzufügen"
+        ></input>
```

Comment:
Very Nice!  I don't know if I understood it correctly, but After adding a new todo, the search field might empty again. Add “Event..target.value“in your add items Function.

Reply:
Sorry, i mean event.target.value = ““;